doc/source/train/distributed-pytorch/configuration.rst

```diff
@@ -1,10 +1,39 @@
 .. _train-config:
 
-Ray Train Configuration User Guide
-==================================
+Common configuration
```

Comment:
Similar to Angelina's comment, I think we should break this up. Right now it feels quite a bit overlapped with the API references themselves.

1. `storage_path` -> We should have a Configuring storage guide.
2. `ScalingConfig` -> This should go in the Scale guide.
3. `RunConfig` -> There probably isn't one page where this goes, but for each functionality that it supports we should show how to use it.
4. `FailureConfig` -> The fault tolerance guide should be extended to be around general failures.
5. `CheckpointConfig` -> This should go in the Checkpoint guide.
6. `SyncConfig` -> Not actually sure off the top of my head, since we're modifying/removing some of the fields.

Reply:
Yeah we can do this in a separate PR. I left this comment here since I was primarily thinking about the overall structure of the documentation, and wanted to see if it even makes sense to get rid of this page.